gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManager.java

```diff
@@ -27,5 +34,134 @@
 
   public static final String DEFAULT_CONTENT_TYPE = "application/octet-stream";
 
-  // TODO(ajaykannan): Fix me! Add in missing methods.
+  public class ListOptions {
```

Comment:
After some more experimentation with specifying "fields", I've come across a bit of an issue.  While it seems that a `parent` is by default set to the creator of the project (determined experimentally from the API Explorer), `parent` doesn't seem to be fully supported.  For example, when I try to specify that I only want the field "parent", the server throws an error.  When I add in other fields, the server ignores the field "parent" and returns other requested fields.  This suggests perhaps we shouldn't include parent in our API yet.  On the other hand, a parent seems to be necessary (since the API explorer suggests that it's set by default), so we can't simply take out parent.